analyses/cell-type-ewings/cell_type_annotations/SCPCS000490/SCPCL000822_tumor-cell-validation.Rmd

```diff
@@ -0,0 +1,600 @@
+
+We will look specifically at gains and losses in each chromosome across tumor and normal cells. 
+To do this, we will use the output from `InferCNV` since that output separates out gains and losses in a way that makes it easy for plotting. 
+
+The first heatmap looks at the proportion of each chromosome with a CNV that corresponds to an amplification or gain. 
+Here each row is a cell and each column is a chromosome with the color corresponding to the proportion of the chromosome with a gain detected. 
+
+```{r}
+# need to make a new annotation, since we want to annotate the rows here 
+annotation <- ComplexHeatmap::rowAnnotation(classification = annotation_df$combined_cnv_classification,
+                                            col = list(classification = c("Tumor" = "#00274C", "Normal" = "#FFCB05", "Unknown" = "grey")))
+
+# plot gains in cnv
+plot_cnv_heatmap(classification_df,
+                 cnv_col = "proportion_scaled_dupli",
+                 annotation = annotation,
+                 legend_title = "Copy number \ngain")
+```
+
+Here we see a gain in Chr8 and Chr1 in tumor cells compared to normal cells. 
+We also see other chromosomes with amplifications in tumor cells compared to normal cells and that the tumor and normal cells are nicely separated in the heirarchical clustering. 
+
+Below we make the same plot but with only the losses or deletions. 
+
+```{r}
+# plot loss in cnv
+plot_cnv_heatmap(classification_df, 
+                 cnv_col = "proportion_scaled_loss", 
+                 annotation = annotation, 
+                 legend_title = "Copy number \nloss")
+```
+
+We do not see the expected loss in Chr16, but this is not present in every patient. 
+Additionally, we had already noted the gain in Chr8 in the tumor cells, so there is still at least one expected CNV present. 
+Again, there is a clear separation of tumor and normal cells in terms of presence of CNVs. 
+
+In both of these plots, the majority of "Unknown" cells cluster between tumor and normal or with the normal cells. 
+
+## Create final tumor annotations
+
+In looking at the plots, it looks like there might be some of the "Unknown" cells that should actually be classified as tumor cells. 
+In particular, we expect those that have a gain in Chr8 are tumor cells, so let's just look at Chr8 gains and see if we can find a cut off to use to categorize our "Unknown" cells. 
+
+```{r}
+#density plot of chr8 gains
+ggplot(classification_df, aes(x = proportion_scaled_dupli_chr8, fill = combined_cnv_classification)) +
+  geom_density() +
+  facet_grid(rows = vars(combined_cnv_classification))
+```
+
+It looks like we have a pretty clear distinction between the `proportion_scaled_dupli_chr8` for normal and tumor cells and we could call any cell with a value > 0.25 as tumor cells and those that are less than 0.25 as normal cells. 
```

Comment:
This looks less clear to me, as I think there are some high values in the normal samples: they are just swamped by the many low values.